sdk/trace/sampling.go

```diff
@@ -77,3 +77,10 @@ func NeverSample() Sampler {
 		return SamplingDecision{Sample: false}
 	}
 }
+
+// AlwaysParentSample returns a Sampler that samples only if the parent span is sampled.
```

Comment:
How about:

```suggestion
// AlwaysParentSample returns a Sampler that samples a trace only
// if the parent span is sampled.
```

so it's more in line with docs for other samplers, and have a wrapped line.